server/controllers/request.js

```diff
@@ -0,0 +1,70 @@
+const Request = require("../models/Request");
+const asyncHandler = require("express-async-handler");
+
+// @route GET /requests
+// @desc get requests for logged in user
+// @access Private
+exports.getRequests = asyncHandler(async (req, res) => {
+  const requests = await Request.find({
+    sitterId: req.user.id,
+  });
+
+  res.status(200).json({
+    success: { requests },
+  });
+});
+
+// @route POST /requests
+// @desc Create a new request
+// @access Private
+exports.createRequest = asyncHandler(async (req, res) => {
+  const { sitterId, start, end } = req.body;
```

Comment:
As a bonus, it would nice to validate the parameters from the request body and return a `400` if they're missing

Reply:
I thought lines 30 - 32 were handing this potential issue but apparently I forgot to test it so my mistake